src/software/jetson_nano/display/robot_ui.py

```diff
@@ -35,21 +31,11 @@ class ScreenActions:
 
 # These are the keys for the redis dicationary
 redis_keys = [
-    ROBOT_ID_REDIS_KEY,
-    ROBOT_MULTICAST_CHANNEL_REDIS_KEY,
-    ROBOT_NETWORK_INTERFACE_REDIS_KEY,
-    "battery voltage",
-    "cap voltage",
-    "packet loss",
-    "chip enable",
-    "kick enable",
-    "chip speed",
-    "kick speed",
-    "wheels enable",
-    "fl wheel speed",
-    "fr wheel speed",
-    "bl wheel speed",
-    "br wheel speed",
+    "/robot_id",
+    "/channel_id",
+    "/battery_voltage",
+    "/cap_voltage",
+    "/current_draw",
```

Comment:
nit: We should use the constants defined in `software/constants.h` instead of copying the values.